src/Python/monodomain_2D_HH.py

```diff
@@ -161,16 +161,16 @@
 
 # Read the cellml file either as an argument (useful for testing) or hardcoded text.
 if len(sys.argv) > 1:
-	cellml_file = sys.argv[1]
+	cellmlModel = sys.argv[1]
 else:
-	cellml_file = 'HodgkinHuxley1952.cellml'
+	cellmlModel = 'HodgkinHuxley1952.cellml'
 
 #DOC-START create cellml environment
 # Create the CellML environment
 cellML = iron.CellML()
 cellML.CreateStart(cellMLUserNumber, region)
 # Import a Nobel 98 cell model from a file
-noble98Model = cellML.ModelImport(cellml_file)
+noble98Model = cellML.ModelImport(cellmlModel)

```

Comment:
Not that variable! I was meaning change `noble98Model` to `cellmlModel` the `cellml_file` name is fine.

Reply:
What is wrong with noble98Model? Seems like the variable name is exactly what it is?

Reply:
Except that it's a hodgkin huxley model.

Reply:
It could be any model if passed in through the command line.

Reply:
Ahhhh, point.

Reply:
My bad!! Done.